test/Basics/Labels.js

```diff
@@ -338,6 +338,21 @@ testNoSyntaxErrorWithDanglingStaticAfterLabel=
 testRuntimeErrorWithDanglingStaticAfterLabel=
     `L: static
     x = 0`
+
+testSyntaxErrorWithContinueAfterLabel=
+    `function fn() {
+        L:
+            continue L;
+    }
+    fn1()`

```

Comment:
Is the SyntaxError we're looking for an early syntax error? If so, we shouldn't have to call any function to hit it.

Reply:
Both have been corrected.